log-cleanup/airflow-log-cleanup.py

```diff
@@ -175,20 +175,23 @@
     echo ""
     echo "Running Cleanup Process..."
 
-    FIND_STATEMENT="find ${BASE_LOG_FOLDER}/*/* -type f -mtime \
+    # Delete log files
+    FIND_STATEMENT="find ${BASE_LOG_FOLDER} -type f -mtime \
```

Comment:
```suggestion
    FIND_STATEMENT="find ${BASE_LOG_FOLDER}/* -name ""*"" -type f -mtime \
```

Reply:
this is enough to fix the issue. And `find` statement for empty folders should be fine

Reply:
You may commit this change if you want, but I see no difference there. The `-name` only adds a check which should always be true. Appending the `/*` should also have no effect, as the `-type f` check will ensure, we only delete files anyway.

We run this patch internally for quite some time now without any issues.
Deleting around 8k files a day.

Reply:
> FIND_STATEMENT="find ${BASE_LOG_FOLDER}/* -name ""*"" -type f -mtime \

Thanks a lot!